packages/web-components/src/components/footer/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,178 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import corsproxy from '../../../../../../docs/cors-proxy.md';
+
+# Footer
+
+> The footer component is a required navigational pattern for IBM.com that
+> displays consistently at the bottom of each page.
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/footer)
+> example implementation.
+
+## Getting started
+
+### Quick start
+
+`<dds-footer-container>` automatically renders contents of footer from default service data. If you want change in services/data integration, etc., see ["Customizing data integration" section](#customizing-data-integration) below.
+
+##### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/footer/footer-container';
+```
+
+##### HTML
+
+```html
+<dds-footer-container></dds-footer-container>
+```
+
+### Customizing service/data integration
+
+For better flexibility in service integration, we have `<dds-footer-composite>` that automatically renders contents of footer from the given data.
+You can populate the data automatically from `@carbon/ibmdotcom-servies` by connecting a Redux store into `<dds-footer-composite>`.
+
+The default Redux store as well as its property/action mappers are available in `@carbon/ibmdotcom-web-components/es/components/footer/footer-container` module. You can connect `<dds-footer-composite>` to the default Redux store and its property/action mappers by below code:
+
+
+##### JS
+
+```javascript
+import ConnectMixin from '@carbon/ibmdotcom-web-components/es/globals/mixins/connect';
+import DDSFooterComposite from '@carbon/ibmdotcom-web-components/es/components/footer/footer-composite';
+import { store, mapStateToProps, mapDispatchToProps } from '@carbon/ibmdotcom-web-components/es/components/footer/footer-container';
+
+// Connects `<dds-footer-composite>` to `@carbon/ibmdotcom-servies` and creates `<app-footer-container>`
+customElements.define('app-footer-container', class extends ConnectMixin(store, mapStateToProps, mapDispatchToProps)(DDSFooterComposite) {})
+```
+
+You can replace the Redux store or its property/action mappers of your own, as long as the comnination of those yields to the same data structure as the default one.
+
+##### HTML
+
+```html
+<app-footer-container></app-footer-container>
```

Comment:
let's use the prefix `myapp-` to show in instanced where the container is rolled on their own.

Reply:
Makes sense, done.